lib/Message/Passing/Redis/Role/HasAConnection.pm

```diff
@@ -1,19 +1,30 @@
 package Message::Passing::Redis::Role::HasAConnection;
 use Moo::Role;
 use Message::Passing::Redis::ConnectionManager;
+use MooX::Types::MooseLike::Base qw/ Int /;
 use namespace::clean -except => 'meta';
 
 with qw/
     Message::Passing::Role::HasAConnection
     Message::Passing::Role::HasHostnameAndPort
 /;
-
+has reconnect => (
+    isa => Int,
+    is  => 'ro',
+    default => sub { 0 },
+);
+has every => (
+    isa => Int,
+    is => 'ro',
+    default => sub { 0 },
+);
 sub _default_port { 6379 }
 
+use Data::Printer;
```

Comment:
I feel ashamed, I forgot this debug line here :( I'll submit a new PR to remove it. So sorry!